libclc/clc/lib/generic/common/clc_smoothstep.cl

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
I wasn't sure if the AMD copyright still applies to this copied file, so I kept it.